.github/workflows/variables.yml

```diff
@@ -25,7 +25,7 @@ on:
         value: ${{format('{0}.{1}{2}', jobs.parse.outputs.release, jobs.parse.outputs.patch, jobs.parse.outputs.dash-name)}}
       branch:
         description: Release branch name (release/A.B.x[-N])
```

Comment:
```suggestion
        description: Release branch name (release-A.B.x[-N])
```

Reply:
Done in [cbde0d2](https://github.com/stackrox/stackrox/pull/2343/commits/cbde0d2c06f6b2b8a860c2a55c684a832fb50928)